client/stories/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import Button from "../src/common/Components/Button/Button";
+
+storiesOf("Button", module).add("simple usage", () => (
```

Comment:
To follow React's component principle, it's common to keep .stories files in the appropriate component's folder alongside the actual source .js file.
This way you'll have potentially more reusable components architecture that is working as a plugable piece with a default storybook config.